CHANGELOG.md

```diff
@@ -3,13 +3,18 @@
 ## Unreleased
 
 ### Breaking Changes
-...
+ - Changed the behavior of `IncrementalEvaluator::do_evaluate`  such that it now promotes all output to `do_incremental_evaluate`instead of only the new outputs.
```

Comment:
```suggestion
 - Changed the behavior of `IncrementalEvaluator::do_evaluate` such that it now sends all output to `do_incremental_evaluate` instead of only the new outputs.
```
Is output an actual `Output` ? otherwise its not quite clear what happens here

Reply:
Output is `SuccessfulExampleOutput`. Adapted the text accordingly.